draft-ietf-tls-rfc8446bis.md

```diff
@@ -914,8 +914,8 @@ as for a 1-RTT handshake with PSK resumption.
 IMPORTANT NOTE: The security properties for 0-RTT data are weaker than
 those for other kinds of TLS data.  Specifically:
 
-1. This data is not forward secret, as it is encrypted solely under
-keys derived using the offered PSK.
+1. This data is not necessarily forward secret, as it is encrypted solely under
+keys derived using the offered PSK; see {{single-use-tickets}}.
```

Comment:
In light of my comment on the linked issue, I'd suggest a phrasing more akin to:

The client in general must assume that this data is not forward secret: it is encrypted solely under keys derived using the offered PSK and the presented ticket inherently must allow for recovery of that PSK.  In specific circumstances, with a cooperating server, some level of forward secrecy may be provided for this data; see {{single-use-tickets}}.

Reply:
I pushed a new phrasing, hope this is better?
(The new phrasing tries to give a high-level answer as to "forward secret, yes or no", and avoids mentioning the PSK for conciseness.)